src/domain/Reward.ts

```diff
@@ -28,8 +29,7 @@ export function loadOrCreateRewardDayTotal(date: BigInt): RewardDayTotal {
     rewardDayTotal = new RewardDayTotal(date.toString())
     rewardDayTotal.todayValue = BigInt.fromI32(0)
     rewardDayTotal.toDateAggregateValue = BigInt.fromI32(0)
-    // 0.0042 RAD/DAI up to 1M RAD
-    rewardDayTotal.rewardRate = BigDecimal.fromString(initialRewardRate)
+    rewardDayTotal.rewardRate = BigDecimal.fromString('0')
```

Comment:
Why set the rate to 0?

Reply:
We will likely update the logic by which we set the reward rate over time thus initializing this with the correct value would lead to code for the reward logic to be scattered in the code-base. In the future the reward logic will look something like this:

```
if (block < 1000) { 
    // initial reward logic
} else if (block < 2000) { 
   // updated reward logic
} else if (block < 3000)
  // more updates
} else {
   // current reward logic
}
```